tests/copp/conftest.py

```diff
@@ -27,6 +27,6 @@ def pytest_addoption(parser):
         "--copp_swap_syncd",
         action="store",
         type=bool,
-        default=True,
+        default=False,
```

Comment:
I have ran test_copp CLI command with `--copp_swap_syncd=False`. "--copp_swap_syncd" was "True". If we want to change "--copp_swap_syncd" parameter to another value, we add  `--copp_swap_syncd` to CLI command of running test. But if we define this param to "**False**" we get "**True**", because "--copp_swap_syncd" parameter has bool type. I have changed this parameter, because with older changes "--copp_swap_syncd" doesn't have a sens.

Reply:
ack, good catch!